demos/redactions.yaml

```diff
@@ -0,0 +1,23 @@
+doc: |
+  An example of using redactions.
+bindings:
+  '?X_NEVERSAY': 'I love crêpes.'
+spec:
+  phases:
+    phase1:
+      steps:
+        - '$include<include/mock.yaml>'
+        - pub:
+            payload: '{"do not say it":"?X_NEVERSAY"}'
+        - recv:
+            pattern: '{"do not say it":"?*x"}'
+        - run: |
+            redactRegexp('[nN]i!');
+            guard:
+              return 0 <= bs['?*x'].indexOf("crêpes")
```

Comment:
I had to align the guard  to where the `run` is defined for the test to register the guard here.

Reply:
Thanks! Fixed (to where the source that was a guard was moved to a `run` where it belonged).